docsteady/cycle.py

```diff
@@ -95,7 +95,10 @@ def extract_custom_fields(self, data):
         # Custom fields
         custom_field_values = data.get("customFieldValues", list())
         for custom_field in custom_field_values:
-            string_value = custom_field["stringValue"]
+            if ("booleanValue" in custom_field.keys()):
```

Comment:
Seems like an extra unneeded layer of parentheses.  Also, you shouldn't need the `.keys()`; `in` for a dictionary is already testing only the keys.

Is there a possibility that both exist?  Should the boolean always take precedence over the string if so?

Reply:
I think the underlying content can always return a string value except when its a boolean - they do not exist together.  I though I needed to keys since i want to check the KEY exists not the Value ..  I can try removing keys and of course paranthises

Reply:
ok done and still working